docs/buildah-inspect.md

```diff
@@ -1,14 +1,14 @@
 # buildah-inspect "1" "May 2017" "buildah"
 
 ## NAME
-buildah\-inspect - Display information about working containers or images.
+buildah\-inspect - Display information about working containers or images or manifest lists.
 
 ## SYNOPSIS
 **buildah inspect** [*options*] [**--**] *object*
 
 ## DESCRIPTION
-Prints the low-level information on Buildah object(s) (e.g. container, images) identified by name or ID. By default, this will render all results in a
-JSON array. If the container and image have the same name, this will return container JSON for unspecified type. If a format is specified,
+Prints the low-level information on Buildah object(s) (e.g. container, images, manifests) identified by name or ID. By default, this will render all results in a
+JSON array. If the container and image or manifest have the same name, this will return container JSON for unspecified type. If a format is specified,
```

Comment:
```suggestion
JSON array. If the container, image, or manifest have the same name, this will return container JSON for an unspecified type. If a format is specified,
```

Reply:
"manifest list", or "list", please, plural if need be.